### ISSUE TITLE
zoom in on all component plots simultaneously

### ISSUE THREAD

**Issue by cafinn-usgs** [usgs/geomag-plots#57]
Rather than only being able to zoom in on an individual trace, it would be useful to zoom in on all displayed traces at the same time, using the wheel on the mouse.

**Comment by emcwhirter-usgs** [comment]
Is there a use for zooming in on a single plot as well? Does this also mean that all plots have to be at the same scale in the first place?

**Comment by cafinn-usgs** [comment]
Yes and no.  Yes, there is *possibly *still a use for zooming in on a
single plot.  No, all the plots wouldn't necessarily have to be at the same
vertical scale in the first place.  The intent is just to be able to zoom
in on the same time scale for all of the traces at the same time.

In theory, you should be able to do both in two steps:  set the same
vertical scale first, and then zoom in on all of the traces.

I say 'possible' for the first item, because it may turn out that once a
user can zoom in on all of the traces at the same time, there might not be
much use for zooming in on individual traces.

**Comment by ttwhite-USGS** [comment]
Also enable panning on all plots simultaneously.

**Comment by hasimpson-usgs** [comment]
Make a pan or zoom on any plot will update the application model time extent.
Plots will display the current application model time extent.
